Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyExport.Codeunit.al

```diff
@@ -36,6 +37,7 @@ codeunit 30284 "Shpfy Company Export"
         Shop: Record "Shpfy Shop";
         CompanyAPI: Codeunit "Shpfy Company API";
         CatalogAPI: Codeunit "Shpfy Catalog API";
+        ShpfySkipRecord: Codeunit "Shpfy Skip Record";
```

Comment:
The naming of the variable is inconsistent. Let's use SkippedRecord (after renaming the codeunit).

Rule: Lose Shpfy or Shopify if same name does not exist in Base Application. Add Shopify if it exists.

Example:

"Shpfy Shop" -> Shop
"Shpfy Customer" -> ShopifyCustomer